renderer/csv.go

```diff
@@ -0,0 +1,124 @@
+package renderer
+
+import (
+	"bytes"
+	"encoding/csv"
+	"fmt"
+
+	"github.com/ONSdigital/dp-table-renderer/models"
+	"github.com/go-ns/log"
+)
+
+// RenderCSV returns a csv representation of the table generated from the given request
+func RenderCSV(request *models.RenderRequest) ([]byte, error) {
+	var buf bytes.Buffer
+	writer := csv.NewWriter(&buf)
+
+	model := createModel(request)
+
+	err := writeTitles(writer, request)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: write units
```

Comment:
Can this comment be removed or do you still need to write some tests?

Reply:
'units' actually refers to the units used in the table. I'm waiting on a decision from Benjy as to how to present them.